src/views/Settings.vue

```diff
@@ -318,6 +321,15 @@ export default {
     localEdgeAddress () {
       this.edgeAddress = undefined
       this.$store.dispatch(REMOVE_REMOTE_PEER_ID)
+    },
+    async fetchEdgeDetails () {
+      try {
+        const details = await this.edgeAPI.getEdgeStatus()
```

Comment:
@vickywane I just realized that this code does not handle older Edge device versions. If the status API doesn't exist or is not available, this code will through undesirable exceptions.